source/tests/unit_test/test_api.py

```diff
@@ -406,6 +407,27 @@ def test_describe_dataset(mock_session_response, test_configs):
         assert response.json_body == expected_response
 
 
```

Comment:
I had to add this test in order to meet the 80% coverage requirement in sonarqube.